torch_geometric/explain/explanation.py

```diff
@@ -129,3 +131,136 @@ def _apply_masks(
             out = out.subgraph(node_mask)
 
         return out
+
+
+class HeteroExplanation(HeteroData):
+    r"""Holds all the obtained explanations of a heterogeneous graph.
+
+    The explanation object is a :obj:`~torch_geometric.hetero_data.HeteroData`
+    object and can hold node-attribution, edge-attribution,
+    feature-attribution. It can also hold the original graph if needed.
+
+    Args:
+        node_mask (Dict[NodeType, Tensor], optional): Dictionary of Node-level
+            mask with shape :obj:`[num_nodes]`. (default: :obj:`None`)
```

Comment:
```suggestion
        node_mask (Dict[NodeType, Tensor], optional): Dictionary of Node-level
            mask with  key `NodeType` and value `Tensor` of 
            shape :obj:`[num_nodes]`. (default: :obj:`None`)
```
Nitpick: just to be clear that the shape of the dict isn't `num_nodes`.